config/default.json

```diff
@@ -1,8 +1,7 @@
 {
   "autoApproveTokenTransfers": true,
   "defaultVotingMachine": "AbsoluteVote",
-  "gasLimit_deployment": 5700000,
-  "gasLimit_runtime": 4543760,
+  "foundersInGasLimitArc": 3,
```

Comment:
why does this need to be configurable if you cant configure the gas limits themselves?

Reply:
'foundersInGasLimitArc' should not still have been in there.  I have removed it.

This config file no longer relates to gas limits at all, `gasLimit.js` has replaced it in that respect, and technically one can modify that file to change the gas limits, though it is not as handy as environment variables.  I don't see much need for customizing gas limits so I think removing the ability won't be missed.